### DISCUSSION TITLE
Code owners review and approve only code files owned by them

### DISCUSSION THREAD

**Discussion by erwinka** [community/community#21834] [New to GitHub]
<p>Hello Masters,<br>
How my team that is owner of specific files in master could review only those files in a PR includes also other files are not owned by my team w/o approving the whole PR but only the files signed for my team as code owner?<br>
Thank you for your fast turnaround response.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! What you want to do is enforce that the code owner approves the PR before it can be merged. Check out here: <a href="https://help.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests" rel="nofollow noopener">https://help.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests</a> for a step by step guide as to how to do that. </p>

**Reply by Starttoaster** [reply]
I'm not sure that's a complete answer. I presume that the context of the question is that there are files that aren't "owned" by anyone via CODEOWNERS, and they would like their review to only count for the code that they are CODEOWNERS for, and not for un-owned code in the pull request. As far as I know, currently, their review would count for both in that scenario.
